tests/vanilla/basic.test.ts

```diff
@@ -70,6 +70,13 @@ it('can get the store', async () => {
   expect(store.getState().getState2().value).toBe(1)
 })
 
+it('can get the initial state', () => {
+  const initial = { value: 1 }
+  const store = createStore(() => initial)
+
```

Comment:
Let's add this so that it tests if getInitialState does return the initial one.
```suggestion
store.setState({ value: 2 })
```